### DISCUSSION TITLE
when build ,how to embed many files?

### DISCUSSION THREAD

**Discussion by tu6ge** [tauri-apps/tauri#4679] [Q&A] [answered]
I am building an app with unity3d , deveplemnt environment is ok, but unity has a few of exe file， and some dll file. so, I can not build in app .

**Comment by tu6ge** [comment, marked as answer]
My problem solved

**Reply by ARYAN0KUMAR** [reply]
how did you solve that probem?

**Reply by FabianLars** [reply]
Probably with resources https://tauri.app/v1/guides/building/resources or something completely custom (like downloading files on demand after installation)

**Reply by tu6ge** [reply]
yes, I used `resources`
